prime-router/gradle.properties

```diff
@@ -1,3 +1,8 @@
 # Set the character encoding when running from Gradle.
 # This affects the unit tests and runs of the primeCLI from Gradle.
 org.gradle.jvmargs=-Dfile.encoding=UTF8 -Xmx2048M
+# flyway v9 causes the migrate task the lock up because concurrent index
+# creation cannot happen inside a transaction and flyway v9 uses a new transactional model
+# for postgresql. I tried several different ways to get this to work and this was the
+# only one that was reliable and easy to implement for all engineers
+flyway.postgresql.transactional.lock=false
```

Comment:
This is what works. Putting the command into the `gradle.properties` file allows it pass it down to flyway and voila it runs!